src/commonTest/kotlin/kotools/types/number/StrictlyPositiveIntTest.kt

```diff
@@ -57,20 +62,56 @@ class StrictlyPositiveIntCompanionTest {
 
 class StrictlyPositiveIntTest {
     @Test
-    fun number_toStrictlyPositiveInt_should_pass_with_a_strictly_positive_Number() {
+    fun toStrictlyPositiveInt_should_pass_with_a_strictly_positive_Number() {
         val number: Number = StrictlyPositiveInt.random().toInt()
         val result: Result<StrictlyPositiveInt> = number.toStrictlyPositiveInt()
         result.getOrThrow().toInt() shouldEqual number
     }
 
     @Test
-    fun number_toStrictlyPositiveInt_should_fail_with_a_negative_Number() {
+    fun toStrictlyPositiveInt_should_fail_with_a_negative_Number() {
         val number: Number = NegativeInt.random().toInt()
         val result: Result<StrictlyPositiveInt> = number.toStrictlyPositiveInt()
         assertFailsWith<IllegalArgumentException>(block = result::getOrThrow)
             .shouldHaveAMessage()
```

Comment:
- We should use the `shouldFailWithIllegalArgumentException` function instead of the `assertFailsWith` one.
- We should check the error message.